circle.yml

```diff
@@ -0,0 +1,14 @@
+machine:
+  services:
+    - docker
+
+dependencies:
+  override:
+    - docker info
+    - docker pull lucasgandel/slicerexecutionmodel
```

Comment:
Excellent!  Thank you @jcfr!

On Mon, Sep 28, 2015 at 5:20 PM, Jean-Christophe Fillion-Robin <
notifications@github.com> wrote:

> In circle.yml
> <https://github.com/Slicer/SlicerExecutionModel/pull/46#discussion_r40607070>
> :
>
> > @@ -0,0 +1,14 @@
> > +machine:
> > +  services:
> > +    - docker
> > +
> > +dependencies:
> > +  override:
> > +    - docker info
> > +    - docker pull lucasgandel/slicerexecutionmodel
>
> I created a slicer organization and a repository named
> slicerexecutionmodel.
>
> User lucasgandel should have admin access to that repo.
>
> See https://hub.docker.com/r/slicer/slicerexecutionmodel/
>
> Cc: @thewtex <https://github.com/thewtex> @LucasGandel
> <https://github.com/LucasGandel>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Slicer/SlicerExecutionModel/pull/46/files#r40607070>.
>